update.cf

```diff
@@ -31,6 +29,64 @@ body common control
 }
 
 #############################################################################
+
+bundle agent mpf_update
+# @brief Masterfiles Policy Framework Update Policy
+#
+# @description These policies are related to ensuring the environment is
+# configured properly before a policy run. This includes publishing policy with
+# the masterfiles-stage tooling, making sure that $(sys.inputdir) is updated
+# from its upstream masterfiles, updating CFEngine packages, and ensuring the
+# necessary CFEngine components are running,
+{
+  vars:
+      "sorted_update_bundlesequence_end"
+        meta => { "defvar" },
+        slist => sort( @(def.update_bundlesequence_end), "lex" );
+
+      "num_custom_bundles" int => length( "sorted_update_bundlesequence_end" );
+
+  classes:
+      # TODO After 3.10 is no longer supported compress into a single promise
+      # nesting functions
+      "_have_custom_update_bundles"
+        expression => isgreaterthan( $(num_custom_bundles), 0);
```

Comment:
Also `some(def.update_bundlesequence_end, '.*')` will test if a list is not empty.